src/components/dialogs/parameters/shortcircuit/short-circuit-parameters.tsx

```diff
@@ -147,12 +168,16 @@ const ShortCircuitFields: FunctionComponent<ShortCircuitFieldsProps> = ({
                 PREDEFINED_PARAMETERS.ICC_MAX_WITH_CEI909 &&
             watchInitialVoltageProfileMode === INITIAL_VOLTAGE.CEI909;
 
-        if (isNominal || isCEI909) {
+        if ((isNominal || isCEI909) && isFeaturesDefaultConfiguration) {
```

Comment:
isNominal should be something like isNominalDefaultConfiguration
and
isCEI909 -> isCEI909DefaultConfiguration

Reply:
Done.